README.md

```diff
@@ -1,62 +1,173 @@
 
-2. to extract the message contained in the proof you should do `node ./lib/cli [path to proof] -s [path to output file]` if the proof is valid the tool will print on the standard output the ParsedProof (the format is mentioned above), will save the proof in the specified path and it will exit with status code 0, if not it will exit with status code different than 0. When the message contained in the proof is a string the message will be written on the file as a UTF-8 string when is `{type: 'hex', value: string}`, the value will be written as binary data.
+### :page_with_curl: _Instructions_
 
-## Embed in a node app:
+For using the Oraclize Proof Verification Tool from the _command line_, execute the following steps:
 
-First clone the repository, install the deps `yarn install` and build the project `yarn build`. The target is ECMA 2015 but if you want to use yarn you should have at least node 4.8.0
+**1)** Clone the repository:
 
-Then you can just import the module in your app with:
-```
-import {verifyProof, getProofType} from 'path to proof verification tool directory' + '/lib/index.js\'
-```
+__`❍ git clone https://github.com/oraclize/proof-verification-tool.git`__
+
+**2)** Install the deps:
+
+__`❍ cd proof-verification-tool && yarn install`__
+
+**3)** Build the project:
+
+__`❍ yarn build`__
+
+#### :mag_right: Proof Verification
+
+When you use the `proof-verification-tool` from the command line, you can check if the proof is valid or extract the message contained in the proof:
+
+**4a)** Check the proof validity:
+
+__`❍ node ./lib/cli [path to proof]`__
+
+  * If the _proof is valid_, the tool prints on the standard output the ParsedProof (format above), exits with status code 0, and shows **SUCCESS** message;
+
+  * If the _proof is unvalid_, the tool exits w/ status code different than 0, and shows **FAILURE** message.
+
+**4b)** Extract the message contained in the proof:
+
+__`❍ node ./lib/cli [path to proof] -s [path to output file]`__
+
+  * If the _proof is valid_, the tool prints on the standard output the ParsedProof (format above), saves the proof in the specified path, and exits with status code 0;
+
+  * If the _proof is unvalid_, the tool exits w/ status code different than 0. When the message contained in the proof is a string, the message will be written on the file as a UTF-8 string when is `{type: 'hex', value: string}`, the value will be written as binary data.
+
+&nbsp;
+
+## Embed in a Node App
+
+For using the Oraclize Proof Verification Tool from a _Node app_, execute the following steps:
+
+**1)** Clone the repository:
+
+__`❍ git clone https://github.com/oraclize/proof-verification-tool.git`__
+
+**2)** Install the deps:
+
+__`❍ cd proof-verification-tool && yarn install`__
+
+**3)** Build the project:
+
+__`❍ yarn build`__
+
+**4)** Import the module in your app with:
+
+__`❍ import {verifyProof, getProofType} from 'path to proof verification tool directory' + '/lib/index.js\'`__
```

Comment:
```suggestion
__`❍ import { verifyProof, getProofType } from 'path to proof verification tool directory' + '/lib/index.js\'`__
```

Fixes linting :P

Have you verified this import style works? I don't use the `import/from` myself so I can't tell just by looking...

Reply:
This code has not been edited, I am assuming this is working as expected

Reply:
You know what they say about assumptions...